core/templates/services/autoplayed-videos.service.ts

```diff
@@ -36,7 +36,7 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class AutoplayedVideosService {
-  autoplayedVideosDict: {} = {};
+  autoplayedVideosDict: any = {};
```

Comment:
We do not allow `any` in our codebase, you must correctly specify the type.

Reply:
I think key/value structure will be used here. Will that be fine?

Reply:
Yes